README.md

```diff
@@ -1,38 +1,51 @@
-| `CORTEX_ADDRESS`   | URL address for the target Cortex cluster                                                                                                                                                                                                  | `true`   | N/A     |
-| `CORTEX_TENANT_ID` | ID for the desired tenant in the target Cortex cluster                                                                                                                                                                                     | `true`   | N/A     |
-| `CORTEX_API_KEY`   | Optional password that is required for password protected Cortex clusters. An encrypted github secret is recommended. https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets | `false`  | N/A     |
-| `ACTION`           | Action to take when syncing rules. Either `diff` or `sync`.                                                                                                                                                                                | `false`  | `diff`  |
-| `RULES_DIR`        | Comma-separated list of directories to walk in order to source rules files.                                                                                                                                                                | `false`  | `./`    |
+| `CORTEX_ADDRESS`   | URL address for the target Cortex cluster                                                                                                                                                                                                  | `false`  | N/A     |
+| `CORTEX_TENANT_ID` | ID for the desired tenant in the target Cortex cluster                                                                                                                                                                                     | `false`  | N/A     |
+| `CORTEX_API_KEY`   | Optional password that is required for password protected Cortex clusters. An encrypted [github secret](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets ) is recommended. | `false`  | N/A     |
+| `ACTION`           | Which action to take. One of `lint`, `prepare`, `check`, `diff` or `sync`                                                                                                                                                                  | `true`   | N/A     |
+| `RULES_DIR`        | Comma-separated list of directories to walk in order to source rules files                                                                                                                                                                 | `false`  | `./`    |
 
 ## Actions
 
+All actions will crawl the specified `RULES_DIR` for Prometheus rules and alerts files with a `yaml`/`yml` extension.
```

Comment:
RULES_DIR or file, you can specify just one right?

Reply:
With cortextool, you can specify any, but with the action, it is always a directory.